installer/dev/project_reunion_definitions.h

```diff
@@ -39,15 +39,56 @@
 #define PR_FRAMEWORK_ARM64_RCTYPE PACKAGE
 #define PR_FRAMEWORK_ARM64_PATH "..\\test\\testpackages\\framework_arm64.msix"
 
+// x86 Main
+#define PR_MAIN_X86_ID L"PR_MAINPACKAGE_X86"
```

Comment:
NIT: MixedCase would be easier to read than ALLUPPERCASE e.g. `ProjectReunion_MainPackage_x86` and `ProjectReunion_DdlmPackage_x86`. Or is there a reason I'm missing for uppercase?

Reply:
I followed existing pattern in resource scripts.